dist/src/main/java/io/camunda/commons/job/JobHandlerConfiguration.java

```diff
@@ -27,8 +27,8 @@
 
 @Configuration(proxyBeanMethods = false)
 @ConditionalOnProperty(
-    prefix = "zeebe.broker.gateway",
-    name = "enable",
+    prefix = "camunda.rest",
+    name = "enabled",
     havingValue = "true",
```

Comment:
❌ I feel like we are breaking here something. Can't we support the old configs as well?

Reply:
No, we are not breaking at this point. If the REST API is enabled, it just registers and initiates the job handler bean. Of course, if, in the case of the broker, the embedded gateway is disabled but the REST API enabled, it would create that bean, which, of course, is not nice, but it doesn't break anything because here, it does not expose any endpoints or anything. It just provides a bean that can be consumed by the REST API.